plugins/modules/clusters.py

```diff
@@ -99,7 +105,7 @@ def run_module():
         name=dict(type="str", required=False),
         openshift_version=dict(type="str", required=False),
     )
-    token = os.environ.get("AI_API_TOKEN")
+    token = apitoken.GetToken()
```

Comment:
@agonzalezrh Do you mind referencing the issue number that this PR will close in the PR description? Thanks

Reply:
Sure, updated description. Issue is described here: https://github.com/vjayaramrh/assistedinstaller/issues/80

Reply:
@agonzalezrh Basically this change closes the issue at https://github.com/vjayaramrh/assistedinstaller/issues/58 
#58